server/src/routes/bots/[id]/vote.js

```diff
@@ -32,7 +33,19 @@ module.exports = {
 
       try {
         await incrementVote(id, request.user.id);
-        
+
+        if (bot.webhook.url && bot.webhook.token) {
+          await axios.post(bot.webhook.url, {
+            bot: bot.id,
+            user: request.user.id,
+            type: 'VOTE'
+          }, {
+            headers: {
+              'Authorization': bot.webhook.token
+            }
```

Comment:
You should only add Authorization header when bot.webhook.token is not null in database. Create headers object and manipulate it.

Reply:
bot.webhook.token is a required variable.

Reply:
How do you plan to do validation in schema if token and url is required? Don't make them required.